inc/based_config.php

```diff
@@ -44,6 +44,13 @@
 
 include_once (GLPI_ROOT . "/inc/autoload.function.php");
 
+// Define constants based on server env variables (define using apache SetEnv directive)
+foreach ($_SERVER as $env_var => $env_value) {
+   if (preg_match('/^GLPI_[\w]+$/', $env_var)) {
```

Comment:
Should not we limit (white or blacklist) which constants may be redefined? For example, we could whitelist all constants that can currently be overriden in `inc/define.php` to be used here as well.

Reply:
It is possible indeed.